web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -184,6 +187,30 @@ class SpeakPage extends React.Component<Props, State> {
     return this.state.recordingStatus === 'recording';
   }
 
+  private keyUprerecording = async (event: any) => {
+    let index = null;
+    //for both sets of number keys on a keyboard with shift key
+    if (event.which === 35 || event.which === 49) {
+      index = 0;
+    } else if (event.which === 40 || event.which === 50) {
+      index = 1;
+    } else if (event.which === 34 || event.which === 51) {
+      index = 2;
+    } else if (event.which === 37 || event.which === 52) {
+      index = 3;
+    } else if (event.which === 12 || event.which === 53) {
+      index = 4;
+    }
+
+    if (index !== null) {
```

Comment:
Does this work when the given index doesn't have a recording yet? I'd imagine it doesn't.

Reply:
yes, this works even when it doesn't have a recording yet

Reply:
Huh, I'm positively surprised that it does. The page is more resilient than I thought. Great!